solutions/USACO dec20-cowntagion.mdx

```diff
@@ -2,14 +2,76 @@
 id: usaco-1062
 source: USACO Silver 2020 December
 title: Cowntagion
-author: Kevin Sheng
+author: Kevin Sheng (Java, Python), Tanish Tyagi (C++)
 ---
 
 [Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
 
+We can create a graph based off the roads between the farms which is given as input.
+The fastest way for COWVID-19 to spread to all the farms is for a farm to have superspreader events until 1 cow can be send to every adjacent farm. This would start at the root node, and repeat until COWVID-19 has spread to the entire farm. 
```

Comment:
```suggestion
The fastest way for COWVID-19 to spread to all the farms is for a farm to have superspreader events until 1 cow can be sent to every adjacent farm. This would start at the root node, and repeat until COWVID-19 has spread to all the farms. 
```